Task7/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { LocationService } from './../../services/geolocation';
+import { LoggerService, DevLoggerService } from './../../services/loggerService';
+
+export function loggerFactory() {
+    return process.env.NODE_ENV === 'development'
```

Comment:
Such approach will include  both services in bundle. But actually we need only one. 
Could you please fix it?